src/bindings/python/src/openvino/frontend/pytorch/patch_builtins.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import builtins
+import torch
+
+originals_map = {
+    'divmod': builtins.divmod
+}
+
+
+def patched_divmod(a, b):
+    if isinstance(a, torch.Tensor):
```

Comment:
Please add comment why we need to add this patch

Reply:
Added